docs/reference/docs/reindex.asciidoc

```diff
@@ -203,10 +205,10 @@ POST _reindex
 // CONSOLE
 // TEST[setup:twitter]
 
-If you want a particular set of documents from the twitter index you'll
-need to sort. Sorting makes the scroll less efficient but in some contexts
+If you want a particular set of documents from the `twitter` index you'll
+need to `sort`. Sorting makes the scroll less efficient but in some contexts
```

Comment:
nit: If this is not refering to the verb "sort" but to the dsl command in the following example, I would use "you'll need to use `sort`".

Reply:
Agreed